io.github.Torrent-file-editor.yaml

```diff
@@ -0,0 +1,39 @@
+app-id: io.github.torrent-file-editor
+runtime: org.kde.Platform
+runtime-version: "5.15-21.08"
+sdk: org.kde.Sdk
+
+command: torrent-file-editor
+modules:
+  - name: torrent-file-editor
+    sources:
```

Comment:
source is usually last in the module

Reply:
fixed here: https://github.com/flathub/flathub/pull/3457/commits/69ef24cf14a8e456ca2b1fa93cb076f62e12d3bf